public/components/register_model/model_details.tsx

```diff
@@ -3,22 +3,38 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import React from 'react';
+import React, { useCallback, useRef } from 'react';
 import { EuiFieldText, EuiFormRow, EuiTitle, EuiTextArea, EuiText } from '@elastic/eui';
 import { useController, useFormContext } from 'react-hook-form';
 import { ModelFileFormData, ModelUrlFormData } from './register_model.types';
+import { APIProvider } from '../../apis/api_provider';
 
 const NAME_MAX_LENGTH = 80;
 const DESCRIPTION_MAX_LENGTH = 200;
 const ANNOTATION_MAX_LENGTH = 200;
 
+const isUniqueModelName = async (name: string) => {
+  const searchResult = await APIProvider.getAPI('model').search({
```

Comment:
I'm wondering, if we do `search` by `name`, will it do exact match or fuzzy search?

Reply:
The name parameter isn't fuzzy search. It searched by `name.keyword` and need fully consistent.